src/consensus/pbft/libbyz/Replica.cpp

```diff
@@ -806,33 +812,43 @@ void Replica::send_pre_prepare(bool do_not_wait_for_batch_size)
     next_pp_seqno++;
     LOG_TRACE << "creating pre prepare with seqno: " << next_pp_seqno
               << std::endl;
-    size_t requests_in_batch;
-    ByzInfo info;
+    auto ctx = std::make_unique<ExecTentativeCbCtx>();
     Pre_prepare* pp =
```

Comment:
unique_ptr

Reply:
unfortunately this is a complicated change as ownership of the pre_prepare is complex and too big to change as part of a unrelated change